regress/sql/cypher_subquery.sql

```diff
@@ -73,13 +73,24 @@ SELECT * FROM cypher('subquery', $$ MATCH (a:person)
 												 }
 									RETURN (a) $$) AS (result agtype);
 
---union, no returns, not yet implemented, should error out
+--union, no returns
 SELECT * FROM cypher('subquery', $$ MATCH (a:person)
 									WHERE EXISTS {
```

Comment:
Just curious (because I don't know EXISTS subquery much), how does WHERE EXISTS work if the inner subquery does not return?

Reply:
In this case you commented on, no returns means that there is no RETURN clause specified. The query will still have a result, even if there is no return clause.

I am not sure what you mean by does not return, as all queries should return a result.